src/environment.js

```diff
@@ -126,7 +131,7 @@ export class DeclarativeEnvironmentRecord extends EnvironmentRecord {
   }
 
   // ECMA262 8.1.1.1.2
-  CreateMutableBinding(N: string, D: boolean): Value {
+  CreateMutableBinding(N: string, D: boolean, isGlobal?: boolean): Value {
```

Comment:
Can we make this "isGlobal: boolean = false"?
isGlobal? introduces three states: undefined, true, false, while the "isGlobal: boolean" only has two states.